frontend/src/AddTask.js

```diff
@@ -19,12 +20,13 @@ export const AddTask = () => {
 
   return (
     <form
-      onSubmit={handleSubmit(({ title }) => {
```

Comment:
Did you consider creating a specific component for this form? That may make the code cleaner.

Reply:
What would be the benefit from creating this specific component you mentioned?

Reply:
This is more like a question than a suggestion, but I believe it could lead to a better separation of concerns, getting a UI component apart from the logic, considering that it can get more complex in the future. Not a big deal though.